actions/ci-benchmarking/debug.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
could you remove this file please? I think the generator generated for testing only.